subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -0,0 +1,588 @@
+These behave slightly differently from `compileOnly` and simply ensure that those dependencies aren't packaged in the WAR file.
+However, the dependencies are included on runtime and test runtime classpaths, so use these configurations if that's the behavior you need.
+
+`import`::
+The `import` scope can be used within both `<dependencyManagement>` and `<dependencies>` blocks.
+If you're dealing with the former scenario, read the section on <<migmvn:using_boms,Using bills of materials>>.
++
+The latter scenario has no equivalent in Gradle.
+You can specify a regular dependency on the POM or its library, but the dependencies declared in the `<dependencies>` block will be treated as normal transitive dependencies of the build.
+In other words, those dependencies in the POM will be mapped to the appropriate classpath based on the declared configuration.
++
+For example, imagine you want to use the `groovy-all` POM for your tests.
+It's solely a POM that has its own dependencies listed inside a `<dependencies>` block.
+The appropriate configuration in the Gradle build looks like this:
++
+.Consuming a POM dependency that has only transitive dependencies
+====
+include::sample[dir="userguide/mavenMigration/basic/groovy",files="build.gradle[tags=pom-dependencies]"]
+include::sample[dir="userguide/mavenMigration/basic/kotlin",files="build.gradle.kts[tags=pom-dependencies]"]
+====
++
+The result of this will be that all `compile` and `runtime` scope dependencies in the `groovy-all` POM get added to the test runtime classpath, while only the `compile` scope dependencies get added to the test compilation classpath.
+Dependencies with other scopes will be ignored.
+
+[[migmvn:declaring_repos]]
+=== Declaring repositories
+
+Gradle allows you to retrieve declared dependencies from any Maven-compatible or Ivy-compatible repository.
+Unlike Maven, it has no default repository, so you have to declare at least one.
+In order to have the same behavior as your Maven build, just configure <<repository_types#sub:maven_central,Maven Central>> in your Gradle build, like this:
+
+.Configuring the build to use Maven Central
+====
+include::sample[dir="userguide/mavenMigration/basic/groovy",files="build.gradle[tags=repositories]"]
+include::sample[dir="userguide/mavenMigration/basic/kotlin",files="build.gradle.kts[tags=repositories]"]
+====
+
+You can also use the `repositories {}` block to configure custom repositories, as described in the <<repository_types#sub:maven_repo,Repository Types>> chapter.
+
+[[migmvn:excluding_deps]]
+=== Excluding transitive dependencies
+
+Maven builds use exclusions to keep unwanted dependencies -- or unwanted _versions_ of dependencies -- out of the dependency graph.
+These can be tricky to migrate not because Gradle doesn't have good support for such exclusions, but because you really need to understand _why_ an exclusion is in place to migrate it properly.
```

Comment:
It's not true that Gradle doesn't have good support for exclusions. It has excellent support for exclusions, only you should tell it _what you mean_, which leads to different solutions based on the use cases. It can be as easy as adding an `exclude`, or be more complicated like a component metadata rule if you need to patch bad metadata.

Reply:
I think you've just said the same thing in a different way. I'll assume from this that the expression I used might be taken the wrong way. Normally it goes like:

> These can be tricky to migrate not because Gradle doesn't have good support for such exclusions _(it does)_, ...

I was just relying on an implicit "(it does)". I will change the expression though to be clearer and less idiomatic.